controllers/users/chatSearch.js

```diff
@@ -11,25 +11,27 @@ const chatSearch = async (req, res) => {
   const userId = req.userId;
 
   let followedUserSearchQuery = `
-        SELECT B.user_id_follower, B.user_id_followed, * 
-        FROM users A 
-        INNER JOIN user_follow_user B
-        ON B.user_id_follower = :userId AND B.user_id_followed = A.user_id
-        WHERE 
-            A.username ILIKE :query AND 
-            A.user_id != :userId AND
-            A.verified_status != 'UNVERIFIED' AND
-            A.is_anonymous = false
-        LIMIT 5
-        OFFSET :offset`;
+    SELECT B.user_id_follower, B.user_id_followed, * 
+    FROM users A 
+    INNER JOIN user_follow_user B
+    ON B.user_id_follower = :userId AND B.user_id_followed = A.user_id
+    WHERE 
+        A.username ILIKE :query AND 
+        A.user_id != :userId AND
+        A.verified_status != 'UNVERIFIED' AND
+        A.is_anonymous = false AND
+        A.user_id != :adminUserId
+    LIMIT 5
+    OFFSET :offset`;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The SQL query is selecting all columns from both `users` and `user_follow_user` tables. This could lead to a performance issue if these tables have many columns. It's better to specify only the columns you need.
```diff
- SELECT B.user_id_follower, B.user_id_followed, * 
+ SELECT B.user_id_follower, B.user_id_followed, A.username, A.user_id, A.verified_status, A.is_anonymous
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->